react-components/tsconfig.test.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "types": [
+      "node",
+      "vite/client",
+      "vitest/globals",
+      "vitest/importMeta",
+      "vitest",
+      "@testing-library/jest-dom"
+    ]
+  },
+  "files": ["src/global.d.ts"],
+  "include": [
+    "src/**/*.stories.tsx",
+    "src/**/*.test.tsx",
+    "src/**/*.spec.tsx",
+    "src/**/*.stories.ts",
+    "src/**/*.test.ts",
+    "src/**/*.spec.ts",
+    "stories/**/*",
+    "tests/**/*",
+    "vite.config.ts",
+    "playwright.config.ts"
+  ]
```

Comment:
I see the previous tsconfig.json had an `exclude`-block that excluded node_modules, dist and storybook-static. Should we have the same here?

Reply:
According to https://www.typescriptlang.org/tsconfig/#exclude, exclude only excludes things that are included. So f.ex. storybook-static and node_modules are not in the include.